data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1473,45 +1513,80 @@ descriptions:new_tribe {
+            -- TRANSLATORS: Note helptext for an atlantean production site: Gold Spinning Mill
+            note = pgettext("atlanteans_building", "Do not build until you have a steady gold supply."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Gold Spinning Mill
+            lore = pgettext("atlanteans_building", "No, this is not Rumpelstiltskin spinning gold from straw, we need real gold."),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Gold Spinning Mill
+            lore_author = pgettext("atlanteans_building", "Painted on a gold spinning mill without supplies.")
          }
       },
       {
          name = "atlanteans_scouts_house",
          helptexts = {
+            -- TRANSLATORS: Special helptext for an atlantean production site: Scout's House without road connection
             no_scouting_building_connected = pgettext("atlanteans_building", "You need to connect this flag to a scout’s house before you can send a scout here."),
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Scout's House
-            purpose = pgettext("building", "Explores unknown territory.")
+            purpose = pgettext("building", "Explores unknown territory."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Scout's House
+            note = pgettext("atlanteans_building", "Will need fish and bread to have his meal the Atlantean way"),
+            -- TRANSLATORS: Lore helptext for an atlantean production site:  Scout's House
+            lore = pgettext("atlanteans_building", "You do not see everything from your high towers. Sometimes you need someone to find out in person, that’s me!"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site:  Scout's House
+            lore_author = pgettext("atlanteans_building", "A Scout answering a soldier")
          }
       },
 
-      -- Medium
+      -- Medium sized Buildings
       {
          name = "atlanteans_sawmill",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Sawmill
-            purpose = pgettext("building", "Saws logs to produce planks.")
+            purpose = pgettext("building", "Saws logs to produce planks."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Sawmill
+            note = pgettext("atlanteans_building", "Will need at least four woodcutters for a constant supply of planks."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site:  Sawmill
+            lore = pgettext("atlanteans_building", "This vertical axis design of our sawmills makes operation simple, yet powerful. Patent pending."),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site:  Sawmill
+            lore_author = pgettext("atlanteans_building", "Askadus, Inventor")
```

Comment:
```suggestion
            lore_author = pgettext("atlanteans_building", "Askadus, Inventor of the vertikal windmill design")
```

Reply:
Sorry kind of duplicate "vertical axis design", will not take that. unless someone else will vote for it.

Reply:
So how does a reader then know who Askadus was.     
maybe keep my suggestion for the author and delete it from the text. Like "The design of our sawmill..."